File: c++/4_percabangan/SwitchCase.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){

    int inputUser;

    cout << "Masukan start urutan dari 5-9" << endl << endl;
    cin >> inputUser;

    switch(inputUser){
        case 5:
            cout << "berurut dari 5" << endl;
        case 6:
            cout << "berurut dari 6" << endl;
        case 7:
            cout << "berurut dari 7" << endl;
        case 8:
            cout << "berurut dari 8" << endl;
        case 9:
            cout << "berurut dari 9" << endl;
            break;
        default:
            cout << "anda tidak mengurutkan sesuai range yang ada" << endl;
    }
}

/*
Pada switch case kita tidak bisa memasukan eskpresi dengan object. Khususnya object string
*/

File: javaScript/4_percabangan/SwitchCase.js
let inputUser = prompt("Masukan start urutan dari 5-9");

switch(parseInt(inputUser)){
    case 5:
        console.log("berurut dari 5");
    case 6:
        console.log("berurut dari 6");
    case 7:
        console.log("berurut dari 7");
    case 8:
        console.log("berurut dari 8");
    case 9:
        console.log("berurut dari 9");
        break;
    default:
        console.log("anda tidak mengurutkan sesuai range yang ada");
}


/*
Pada switch case kita tidak bisa memasukan eskpresi dengan object. salah satunya object string
*/